deployment/src/main/java/io/quarkus/jgit/deployment/JGitBuildTimeConfig.java

```diff
@@ -46,10 +48,34 @@ interface DevService {
         @WithDefault("quarkus")
         String adminPassword();
 
+        /**
+         * The project repository configuration.
+         */
+        ProjectRepository projectRepository();
+
+        /**
+         * The repository to create when the Dev Service starts.
+         */
+        Optional<List<String>> extraRepositories();
```

Comment:
I think you can skip the `Optional` here. AFAIR an empty list is returned if this is not set?

Reply:
That was also my first thought. I had to add the `Optional` because it complaints otherwise.